### ISSUE TITLE
return detach on every attach, once*

### ISSUE THREAD

**Issue by beckend** [rogierschouten/ts-events#14]
```javascript
const unlistener = myEvent.attach(() => {
  console.log('i just wanna do something')
})

unlistener()
```

there will be no need saving the listener to a variable

**Comment by beckend** [comment]
aslo a `myEvent.detachAll()` would not hurt instead of keeping track of everything manually

**Comment by rogierschouten** [comment]
Good point. Detach all exists: simply call detach() without parameters

**Comment by rogierschouten** [comment]
Ok, attach() and once() now return a detacher function. Published as v3.3.0.  Thank you @beckend and let me know if there are any issues with it

**Comment by beckend** [comment]
thanks
